C#/ds/Reverse_linked_list.cs

```diff
@@ -0,0 +1,90 @@
+using System; 
+  
+class GFG { 
+    static void Main(string[] args) 
```

Comment:
@YashThakare Please don't copy paste code from geeks for geeks. Try to understand it and code it yourself.